code/modules/mob/mob.dm

```diff
@@ -542,6 +548,16 @@
 		return
 
 	face_atom(examinify)
+
+	// Show nearby mobs that we're examining something
+	if(isliving(src) && examinify.loc != src && !is_holding(examinify))
+		for(var/mob/M in viewers(4, src))
+			if(M == src || !M.client || M.is_blind())
+				continue
+			if(M.client.prefs && !M.client.prefs.read_player_preference(/datum/preference/toggle/examine_messages))
+				continue
+			to_chat(M, span_subtle("<b>\The [src]</b> looks at \the [examinify]."))
```

Comment:
Not a huge fan of showing people when we examine absolutely anything, though I'd want to see it in game first before saying that I'm completely against it. If it's too spammy then it can be limited to examining objects worn by people.